Assets/Global/Scripts/Player/Player.cs

```diff
@@ -99,7 +103,7 @@ void Update()
                 : 0.0f;
```

Comment:
probably more stylistic preference, but can this ternary not be written as:
`activeAttackCooldown =
            currentState == states.Attacking" ? ... : ...`
            instead of :
            `activeAttackCooldown =
            currentState.GetType().Name != "AttackingState" ? ... : ...`